interfaces/builtin/content.go

```diff
@@ -235,8 +235,12 @@ func (iface *contentInterface) AppArmorConnectedPlug(spec *apparmor.Specificatio
 			fmt.Fprintf(&buf, "  # Read-write content sharing %s -> %s (w#%d)\n", plug.Ref(), slot.Ref(), i)
 			fmt.Fprintf(&buf, "  mount options=(bind, rw) %s/ -> %s/,\n", source, target)
 			fmt.Fprintf(&buf, "  umount %s/,\n", target)
-			apparmor.WritableProfile(&buf, source)
-			apparmor.WritableProfile(&buf, target)
+			// TODO: The assumed prefix depth could be optimized to be more
```

Comment:
Will this be in a follow-up PR? The resulting '/ rw,', '/snap/ rw,', etc in the testsuite look a little weird.

Reply:
Yes because I didn't write this part yet. We would have to know which variable is being referenced and provide different assumed prefix depth for each of the possible variables.